src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -210,14 +218,15 @@ private void styleFetchedListView(CheckListView<CitationRelationItem> listView)
                     VBox vContainer = new VBox();
 
                     if (entry.isLocal()) {
+                        hContainer.setBackground(Background.fill(Color.LIGHTGREEN));
```

Comment:
Some entry of JabRef's `Base.css` should be used. Do not introduce new colors if not necessary. - You can introduce a css class `existing`  (name it similar to other classes in the CSS) and then use an existing color of JabRef. You will see it in the `Base.css`.